data/LIKE/data.json

```diff
@@ -0,0 +1,22 @@
+{

```

Comment:
This is a new addition?

Reply:
This was renamed from the "❤️" directly which was causing problems with validation